src/logic/proposals/encoding/encoding.test.ts

```diff
@@ -359,7 +360,8 @@ describe('isEvmScriptValid()', () => {
   });
 
   // The data for this proposal ware created locally.
-  it('returns true also for simple ETH transfers', async () => {
+  it('returns true for simple ETH transfers (version 1)', async () => {
```

Comment:
Can we run all of these tests (that have version fixed to `1`) with both version v1 and v2? Right now, all the tests which do not use `newFormData` are tested against version 1.